src/index.js

```diff
@@ -0,0 +1,17 @@
+// import _ from 'lodash';
+import './style.css';
+import UIDisplay from './UIDisplay.js';
+
+const sampleTodoList = [
+  { description: 'Go on a Fishing today', completed: false, index: 0 },
+  { description: 'Shopping by 3pm', completed: false, index: 1 },
+  { description: 'Attend Online Class by 9pm', completed: false, index: 0 },
+];
```

Comment:
- [ ] **Required:** 
![image](https://user-images.githubusercontent.com/27709832/184181232-5491a119-57cb-4e74-a4a0-20f57efb4aaf.png)
- You are required to display the list in the order of their index so that irrespective of where they are on the list, the item with the least index is rendered at the top and the item with the highest index is rendered at the bottom.